test/150_connect_forget_test.sh

```diff
@@ -15,8 +15,8 @@ assert_peers() {
 
 start_suite "Connecting and forgetting routers after launch"
 
-weave_on $HOST1 launch
-weave_on $HOST2 launch
+weave_on $HOST1 launch-router
+weave_on $HOST2 launch-router
```

Comment:
> Let me try weave launch -initpeercount instead.

Interesting results from this - the first `weave launch -initpeercount 2` fails:
````
$ time ./150_connect_forget_test.sh 
Connecting and forgetting routers after launch
Container 016b4593cc2184e2b5794994ffe3c892fc14bb20d14f262cc5b1f15b227c1867 not running.

real	0m23.154s
user	0m0.029s
sys	0m0.044s
````
I discussed this with @bboreham and he pointed me towards #1001 which features weaveDNS dying after twenty seconds due to failure to allocate an IP address to the mDNS interface. The root cause is different - in this case we're lacking quorum so the allocation request blocks - but the effect is the same.

Since this problem already affects `weave launch-dns`, I propose not to fix it here - I'll create a separate issue for it instead.